_data/stage3.yml

```diff
@@ -52,279 +39,166 @@
         this.textContent = this.#x.toString();
       }
     }
-    window.customElements.define('num-counter', Counter);
-
-- title: Class Public Instance Fields & Private Instance Fields
-  id: proposal-class-fields
+    window.customElements.define(&#x27;num-counter&#x27;, Counter);
+  presented:
+    - date: January&#xA0;2019
+      url: >-
+        https://github.com/tc39/notes/blob/master/meetings/2019-01/jan-30.md#private-fields-and-methods-refresher
   has_specification: true
-  description: This proposes a combined vision for public fields and private fields, drawing on the earlier Orthogonal Classes and Class Evaluation Order proposals.
+  description: Lorem ipsum.
   authors:
     - Daniel Ehrenberg
-    - Kevin Gibbons
   champions:
     - Daniel Ehrenberg
-    - Jeff Morrison
-    - Kevin Smith
-    - Kevin Gibbons
   tests:
-    - https://github.com/tc39/test262/issues/1161
-  example: >
-    class Foo {
-        instancePropertyBar = 0;
-        static staticPropertyBar = 0;
-        #privatePropertyBar = 0;
-        static #privatePropertyBar = 0;
-
-        set foo(value) { console.log(value); }
+    - >-
+      https://github.com/tc39/test262/pulls?q=is%3Apr+is%3Aclosed+private+methods
+- title: Class Public Instance Fields &amp; Private Instance Fields
+  id: proposal-class-fields
+  example: |-
+    class X {
+      #foo;
+      method() {
+        console.log(this.#foo)
+      }
     }
   presented:
-    - date: January 2019
-      url: https://github.com/tc39/notes/blob/master/meetings/2019-01/jan-30.md#private-fields-and-methods-refresher
-
-- title: Static class fields and private static methods
-  id: proposal-static-class-features
+    - date: March&#xA0;2019
+      url: >-
+        https://github.com/tc39/notes/blob/master/meetings/2020-03/march-31.md#add-support-for-optionalchainprivateidentifier-in-class-features-proposals
   has_specification: true
-  description: This proposal adds Static public fields, Static private methods and Static private fields
+  description: Lorem ipsum.
```

Comment:
are these intended?

Reply:
Lol, nope. Still need to address this before the PR can come out of draft form.

I explain the problem in [a comment below](https://github.com/tc39/tc39.github.io/pull/211/files/2c203bacf60f0d75b94c308a943e80a19a06b256#r487570821).

Reply:
Ideally, we shouldn't overwrite descriptions if they were entered by hand. There will still be editing work even when this is done.